sites/website/src/docs/resources/cheat-sheet.md

```diff
@@ -0,0 +1,711 @@
+| [fast-tooltip](https://www.fast.design/docs/components/tooltip)                           | `fastTooltip()`                                               | [Tooltip](https://explore.fast.design/components/fast-tooltip)                                                                                        |                                                                                                                                             
+| [fast-tree-view](https://www.fast.design/docs/components/tree-view)                       | `fastTreeItem()` `fastTreeView()`                             | [Tree view](https://explore.fast.design/components/fast-tree-view)                                                                                    |                                                                                                                                      
+---
+### [Integrations](https://www.fast.design/docs/integrations/introduction)
+
+FAST libraries can also be used in combination with a wide variety of existing technologies.
+
+[Angular](https://www.fast.design/docs/integrations/angular)
+
+[ASP.NET](https://www.fast.design/docs/integrations/aspnet)
+
+[Aurelia](https://www.fast.design/docs/integrations/aurelia)
+
+[Blazor](https://www.fast.design/docs/integrations/blazor)
+
+[Ember](https://www.fast.design/docs/integrations/ember)
+
+[React](https://www.fast.design/docs/integrations/react)
+
+[Vue](https://www.fast.design/docs/integrations/vue)
+
+[Webpack](https://www.fast.design/docs/integrations/webpack)
+
+Not seeing an integration for your preferred technology?  Open an issue on [GitHub](https://github.com/microsoft/fast/issues).
+
+---
+## [Building Components](https://www.fast.design/docs/components/getting-started)
+### Setup
+
+To define a custom element:
+
+```ts
+import { FASTElement, customElement } from '@microsoft/fast-element';
+
+@customElement('name-tag') // custom element being created
+export class NameTag extends FASTElement {
+
+}
+```
+
+With this in place, you can now use your name-tag element anywhere in HTML with the following markup:
+
+```html
+<name-tag></name-tag>
+```
+
+---
+
+### [Attributes](https://www.fast.design/docs/fast-element/defining-elements#customizing-attributes)
+
+To add attributes to your HTML element, create properties decorated by the `@attr` decorator:
+
+```ts
+import { FASTElement, customElement, attr } from '@microsoft/fast-element';
+
+@customElement('name-tag')
+export class NameTag extends FASTElement {
+  @attr greeting: string = 'Hello';
+
+  greetingChanged() {
+    this.shadowRoot!.innerHTML = this.greeting;
```

Comment:
Should our examples include setting "innerHTML" given that can have security issues?

Reply:
Might also confuse someone into thinking this is how to render something into a component rather than a template binding.  Maybe the example could emit an event?

Reply:
Relates to my comment above. There's a number of ways to leverage and use - how do we determine what should be highlighted as "the way" in the cheat sheet?

Reply:
@scomea @chrisdholt Thank you - updated!